helm/templates/agent-deployment.yaml

```diff
@@ -67,6 +67,8 @@ spec:
                   fieldPath: spec.nodeName
             - name: MY_NAMESPACE
               value: {{ include "nuvlaedge.namespace" . }}
+            - name: TEST_IMAGE_PULL_POLICY
+              value: {{ .Values.TEST_JOB_ENGINE_PULL_POLICY | default "IfNotPresent" | quote }}
```

Comment:
What if instead of `TEST` we prepend it with `_NE`. `NE` is for NuvlaEdge and `_` is to signify that it's a "hidden" space of vars (a-la dot-files/directories). So,
`_NE_IMAGE_PULL_POLICY`
`_NE_JOB_ENGINE_PULL_POLICY`
I'm open to suggestions. I simply want to avoid mentioning the word `TEST` in the production code :)

Reply:
Ah, that's a good point... I re-pended with "TEST" as I figured it would be us testing things i.e. job-engine. But your suggestion is more consistent. And better logic not to leave things as "TEST" for customer eyes? (haha)